build-system/global-configs/experiments-config.json

```diff
@@ -1,5 +1,12 @@
 {
-  "experimentA": {},
+  "experimentA": {
+    "name": "IntersectionObserverPolyfill",
+    "environment": "AMP",
+    "command": "gulp dist --define_experiment_constant=INTERSECTION_OBSERVER_POLYFILL",
```

Comment:
@rsimha guess this goes back to the conversation about doing multiple builds.

we will need the module build version built here as well.

`gulp dist --esm --define_experiment_constant=INTERSECTION_OBSERVER_POLYFILL && gulp dist --define_experiment_constant=INTERSECTION_OBSERVER_POLYFILL`

Reply:
I added it. @rsimha LMK if you're ok with that.

Reply:
As I said in https://github.com/ampproject/amphtml/pull/27499#discussion_r403213434, this will double build time and substantially increase Travis round-trip time for CI and releases. I don't think we can afford to check in this PR as it is currently written since it will have an effect on the Travis run for every single commit to `master`.

I wonder if there is a valid case for spinning off a separate out-of-band Travis project for experimental builds and somehow plugging it in to our release process. @erwinmombay Can you start off a separate discussion about this?

/cc @ampproject/wg-infra for visibility.

Reply:
I pushed again the version with a single build.

Reply:
As discussed offline, @dvoytenko has reverted the double-build change, and @erwinmombay will  separately consult with @ampproject/wg-infra to come up with a way to test module against no-module without increasing Travis round-trip time for normal development / release.